digiwf-apps/pom.xml

```
@@ -7,7 +7,7 @@
   <parent>
     <groupId>de.muenchen.oss.digiwf</groupId>
     <artifactId>digiwf-core</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.1.2-SNAPSHOT</version>
```

Warum hier 1.1.2? In allen anderen poms steht 1.1.3

```suggestion
    <version>1.1.3-SNAPSHOT</version>
```